test/cpp/end2end/cfstream_test.cc

```diff
@@ -47,8 +47,10 @@
 #include "test/cpp/end2end/test_service_impl.h"
 
 #ifdef GRPC_CFSTREAM
+using grpc::ClientAsyncResponseReader;
 using grpc::testing::EchoRequest;
 using grpc::testing::EchoResponse;
+using grpc::testing::RequestParams;
 using std::chrono::system_clock;
 
 namespace grpc {
```

Comment:
`netmask_` seems never used? :)

Reply:
and since it is loopback address, "127.0.0.2" might be (a tiny bit) better?

Reply:
Thanks for pointing out the unused variable. I'll remove it and use 127.0.0.2 like you suggested.